libs/@guardian/libs/jest.testEnvironment.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/no-default-export -- it's what JSDOM env does */
+
+// https://stackoverflow.com/questions/74945569/cannot-access-built-in-node-js-fetch-function-from-jest-tests
+
+import JSDOMEnvironment from 'jest-environment-jsdom';
+
+export default class FixJSDOMEnvironment extends JSDOMEnvironment {
```

Comment:
This is interesting… is it created as new linting errors started cropping up or the removal of `axios` and `http`?

In the spirit of keeping PR focussed, would it be worth extracting these changes either before or after to bring the right level of attention to them?

Reply:
https://github.com/guardian/csnx/pull/1356